pythonforandroid/distribution.py

```diff
@@ -42,12 +42,19 @@ def __repr__(self):
         return str(self)
 
     @classmethod
-    def get_distribution(cls, ctx, name=None, recipes=[],
-                         ndk_api=None,
-                         force_build=False,
-                         extra_dist_dirs=[],
-                         require_perfect_match=False,
-                         allow_replace_dist=True):
+    def get_distribution(
+            cls,
+            ctx,
+            *,
```

Comment:
Just curious why enforcing keyword arguments and why from here and not from `ctx` for instance?

Reply:
Partly because it's simpler not to break the existing code where ctx is the first argument, and partly because I think it's a fine api style to have ctx by an implicit essential argument that should come first while everything else has no particular order to it. ctx is also a consistent p4a concept with the same name everywhere.

Reply:
OK I understand, thanks for explaining :+1: